gems/smithy/lib/smithy.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'rails/railtie'
+require 'smithy-client'
```

Comment:
Should we require smithy client only when we are generating a client?

Reply:
We will have to require it for types too, and since we have no server, I think this is fine for now. Same for rails/railtie, probably only for server, but active_support for all.